src/gamefile/file.game

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" ?>
+<game
+  type="unvanquished"
+  index="1"
+  name="Unvanquished"
+  enginepath_win32="C:/Program Files/Unvanquished/"
+  enginepath_linux="/usr/lib/unvanquished/"
+  enginepath_macos="/Applications/Unvanquished/"
+  engine_win32="daemon.exe"
+  engine_linux="../../../usr/bin/unvanquished"
+  engine_macos="daemon.app"
+  prefix=".unvanquished"
+  basegame="pkg"
+  basegamename="Unvanquished"
+  unknowngamename="Unvanquished Mod"
+  shaderpath="scripts"
+  archivetypes="pk3"
+  texturetypes="tga jpg png pcx bmp webp"
```

Comment:
I'm not sure if we use webp anywhere in our asset base but in my humble opinion it has only caused trouble for us and we should not advertise its use at the moment anyway.